spec/features/form/check_answers_page_spec.rb

```diff
@@ -133,6 +133,19 @@
       end
     end
 
+    it "has no summary cards if the subsection is not household characteristics" do
```

Comment:
This should probably be phrased in terms of attributes rather than a specific section name

Reply:
Do you have a suggestion for how it might look phrased in terms of attributes? I did it this way since this design only applies to the household characteristics section 🙂

Reply:
I guess the number of cards shown should match the number the definition for the section expects

Reply:
The section definition doesn't define the number of cards shown, it's defined at the question level using `check_answers_card_number`. When you get the applicable questions for a subsection you group them into cards using group by. I wasn't quite sure how to express this test in any other way, maybe something like `does not group questions into summary cards if the questions in the subsection don't have a check_answers_card_number attribute"?